### DISCUSSION TITLE
[dynamic_links] not work on real device

### DISCUSSION THREAD

**Discussion by PhloxDev** [firebase/flutterfire#6195] [Q&A]
when example app run on emulator every thing is ok but on real device _initDynamicLinks_ not call.

Flutter Version:2.2.0

firebase_dynamic_links: ^2.0.3
real device Android Version 5 and 6 , 8 --> not ok

Emulator 9 --> ok

**Comment by nilsreichardt** [comment]
Please open an issue for this.
